.travis.yml

```diff
@@ -5,6 +5,7 @@ language: python
 cache:
     directories:
         - $HOME/.cache/pip
+        - $HOME/ossl-098
```

Comment:
Can we add the letter here, in the event we want to add more in the future I don't want untangling this to be a pain.